JuliaDiff/index.md

```diff
@@ -0,0 +1,83 @@
+- [Tracker.jl](https://github.com/FluxML/Tracker.jl): Operator overloading reverse-mode AD. Most well-known for no longer being [Flux's](https://github.com/FluxML/Flux.jl) AD. But still used in several places about the ecosystem.
+- [AutoGrad.jl](https://github.com/denizyuret/AutoGrad.jl): Originally a port of the python AutoGrad package. Primarily used in [Knet](https://github.com/denizyuret/Knet.jl/).
+
+- [Zygote.jl](https://github.com/FluxML/Zygote.jl): IR-level source to source reverse-mode AD. Very widely used. Particularly notable for use in [Flux](https://github.com/FluxML/Flux.jl). Also features a secret experimental source to source forwards mode AD.
+- [Yota.jl](https://github.com/dfdx/Yota.jl): IR-level source to source reverse-mode AD.
+- [XGrad.jl](https://github.com/dfdx/XGrad.jl): AST level source to source reverse-mode AD. Not currently in active development.
+- [ReversePropagation.jl](https://github.com/dpsanders/ReversePropagation.jl): Scalar, tracing-based source to source reverse-mode AD.
+- [Enzyme.jl](https://github.com/wsmoses/Enzyme.jl): Scalar, LLVM source to source reverse-mode AD. Experimental.
+
+
+### Forward mode
+- [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl): scalar operator overloading forward-mode AD. Very stable. Very well-established.
+- [ForwardDiff2](https://github.com/YingboMa//ForwardDiff2.jl): experimental non-scalar hybrid operator-overloading/source-to-source forward-mode AD. Not currently in development.
+
+### Symbolic:
+- [ModelingToolKit.jl](https://github.com/JuliaDiffEq/ModelingToolkit.jl): A pure Julia computer algebra system. While its docs focus on some particular domain use-case it is a fully general purpose system.
+
+### Exotic
+- [TaylorSeries.jl](https://github.com/JuliaDiff/TaylorSeries.jl) computes polynomial expansions; which is the generalization of forward-mode AD to n-th order derivatives.
+- [NiLang.jl](https://github.com/GiggleLiu/NiLang.jl): Reversible computing DSL, where everything is differentiable, by reversing.
+
+### Finite Differencing
+(yes, we said at the start to stop approximating deriviatives, but these packages are faster and more accurate than you would expect finite differencing to every achieve.)
+
+- [FiniteDifferences.jl](https://github.com/JuliaDiff/FiniteDifferences.jl): High-accuracy finite differencing with support for almost any type (not just the usual arrays and numbers)
+- [FiniteDiff](https://github.com/JuliaDiff/FiniteDiff.jl): High-accuracy finite differencing with support for efficient calculation of spares Jacobians via coloring vectors.
+- [Calculus.jl](https://github.com/JuliaMath/Calculus.jl): Largely deprecated, legacy package. New users should look to FiniteDifferences and FiniteDiff instead.
+
+### Rulesets:
+- [ChainRules.jl](https://www.juliadiff.org/ChainRulesCore.jl/stable/): extensible AD independent rule-set.
```

Comment:
Maybe some way to distinguish ChainRules-ecosystem from just ChainRules.jl?

```suggestion
- [ChainRules](https://www.juliadiff.org/ChainRulesCore.jl/stable/): Extensible, AD-independent rules.
```

Reply:
Ah yes I mean to do that but it got caught in a global find-replace when I added jl to everything